uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/AbsenceType.xml

```diff
@@ -181,9 +181,13 @@
   <field>
     <name>category</name>
     <title>Category</title>
-    <type>int unsigned</type>
+    <type>varchar</type>
+    <length>10</length>
```

Comment:
Length should be 512 also to match the length for the Value column on the civicrm_option_value table.